docs/configuration.md

```diff
@@ -339,23 +339,9 @@ For example, `ruff check /path/to/excluded/file.py` will always lint `file.py`.
 
 ### Default inclusions
 
-By default, Ruff will discover files matching `*.py`, `*.ipy`, or `pyproject.toml`.
+By default, Ruff will discover files matching `*.py`, `*.pyi`, `*.ipynb`, or `pyproject.toml`.
 
 To lint or format files with additional file extensions, use the [`extend-include`](settings.md#extend-include) setting.
-
-=== "pyproject.toml"
-
-    ```toml
-    [tool.ruff]
-    extend-include = ["*.ipynb"]
-    ```
-
-=== "ruff.toml"
-
-    ```toml
-    extend-include = ["*.ipynb"]
-    ```
-
 You can also change the default selection using the [`include`](settings.md#include) setting.
```

Comment:
Any suggestion on what can be use for this example? I'm worried that it _might_ signal to the users that we support that file extension. Using `*.ipynb` was ok because we do support it. This is the reason I've removed this example.

Reply:
`extend-include` has an example with `.pyw` files. Although I think the example isn't needed because we already link to `extend-include` which has an example.

Reply:
Yeah, I think not having an example here is fine.